terraform/deployments/rds/ckan_production_postgres_14_upgrade.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_db_parameter_group" "postgresql14_green_params" {
```

Comment:
Is this parameter group going to stick around? If so, it should probably have something in the name to distinguish it (since its only for CKAN but the name is generic)

Reply:
Only temporarily but yeah the resource name should be `ckan_postgresql14_green_params`

Reply:
I've updated it now to `ckan_postgresql14_green_params`